### ISSUE TITLE
The drop of loss function is very low, but there are many false positives in the actual test

### ISSUE THREAD

**Issue by mujinbao** [AlexeyAB/darknet#4962]
My training set has two categories, with a total of more than 8200 images. I use yolov3-voc.cfg default parameters, and the final loss function can be reduced to about 0.09, but in the actual test, there are many false positives;

On the contrary, I adjusted the learning rate attenuation strategy. When I multiplied 0.1 at 3500 and 5500, the final loss would be about 0.35-0.4. After testing, I reduced a lot of false positives. I didn't do the mAP calculation, but it was obvious through the visual experience. Why is this phenomenon
